client/src/contexts/ItemsContext.jsx

```diff
@@ -138,12 +140,27 @@ export function useItems() {
 
 // Provides the store to your application
 export function ItemsProvider({ children }) {
+  const [items, setItems] = useState('')
+
+  useEffect(() => {
+    fetch('http://localhost:4001/items')
+      .then(res => {
+        return res.json()
+      })
+      .then((data) => {
+        setItems(data)
+      })
+  }, [])
+
   return (
     // wrap our React components with our store provider (store has "knowledge of your app")
-    <ItemsContext.Provider value={items}>
-      {/* eventually "children" will be our react app */}
-      {children}
-    </ItemsContext.Provider>
+    <div>
+      {items && <ItemsContext.Provider value={items}>
```

Comment:
Not sure what the `items &&` is for. Could you explain that to me?

Reply:
Since I didn't know how to fetch data from the server, I just used a reference and copied what they did. I updated this and it's okay now. I sent the link in discord too.